src/index.js

```diff
@@ -22,32 +19,20 @@ class Client {
    */
   constructor (addr) {
     this.multiaddr = addr
-    this.server = null
-    this.socket = new Socket({
-      readable: true,
-      writable: true,
-      allowHalfOpen: true
-    })
-    this.socket.on('error', (_) => {
-      this.close()
-    })
+    this.tcp = new TCP({ upgrader: passThroughUpgrader, allowHalfOpen: true })
```

Comment:
I forgot to remove this from the daemon when I was playing around with the sockets. It's not currently supported by `libp2p-tcp` and we have workarounds.
```suggestion
    this.tcp = new TCP({ upgrader: passThroughUpgrader })
```